YARN_MIGRATION.md

```diff
@@ -0,0 +1,23 @@
+# YARN 2 MIGRATION DOC
```

Comment:
thinking that this file just contains all the TODOs right, should this be in notion instead of in the codebase? Wondering if we should use this migration doc to document why/how we changed to yarn2? No big deal otherwise just curious

Reply:
Putting this stuff in a `Guide to Migrate to Yarn V2` in notion might be a good idea since it'll make it easier for other teams to do the same stuff

Reply:
ya just not sure if this doc should go into the codebase

Reply:
yea 100%

Reply:
yep!